certification.md

```diff
@@ -1,62 +1,54 @@
-   * Checking ARI data in several random dates within one year.
-6. Test ARI delta updates
-   * Mews will change prices and restrictions of connected rates and availability of connected space types and verify all changes are correctly updated in Channel manager.
-7. Channel manager creates test reservation groups.
-   * One single room reservation.
-   * One multi room reservation group
-     * To make sure that group bookings are handled correctly \(i.e. using one ChannelManagerId for the whole group\).
-     * Ideally each reservation in the group should have as much different values as possible \(dates, room type, rate, ...\).
-   * Accuracy of the data is checked.
-8. Channel manager modifies the created test reservations group
-   * Dates change, change of the name of the customer, adding a room into the originally individual reservation.
-   * Changing dates/room type 
-   * Cancelling one reservation \(ideally not the “last” one\) of the originally group booking. \(always by sending “current state” = without the cancelled reservation, do not send any “diff”\).
-9. Channel manager cancels the created test reservations group
+## Pre-Certification
+
+1. Email integrations@mewssystems.com to request a Client token to be set up in the Mews [Test Environment](mews-api.md#test-environment).
+2. Create the following endpoint URLs for the Mews [Test Environment](mews-api.md#test-environment)
+    * [Update Availability](channel-manager-api.md#update-availability) 
+    * [Update Prices](channel-manager-api.md#update-prices)
+    * [Update Restrictions](channel-manager-api.md#update-restrictions)
+    * [Confirm Booking](channel-manager-api.md#confirm-booking) 
+    * [Change Notification](channel-manager-api.md#change-notification) \(optional\)
+3. Email the endpoints to integrations@mewssystems.com.
+4. Set up the connection in the channel manager system.
+5. Use the [Get Properties](mews-api.md#get-properties) call to receive the `connectionToken` for the [Test Hotel](mews-api.md#test-environment).
+6. Pull hotel information, space mapping and rate mapping information using the [Get Configuration](mews-api.md#get-configuration) call.
+7. Map the [Test Hotel](mews-api.md#test-environment) in the UI of the channel manager backend using the data received in the [Get Configuration](mews-api.md#get-configuration) call.
+
+**Note:** Multi-property Mews customers (vacation rentals, apartments, villas, etc.) are often set up in Mews as spaces of one Mews property.  E.g. Property is the Mews Customer and Spaces are the rental properties.  These spaces (properties) can be sent to the channel manager as multiple connections, 1 connection token per space (property) or as a single connection for the entire property manager.
+
+## Certification Tests
+
+These tests will be done during a 60-minute call with the Mews Integrations team and the channel manager.  
+
+| Test                             | Requirements                         |
```

Comment:
I'd format the table differently

| Test | Requirement | Scenario | Expected result | Notes |
| --- | --- | --- | --- |
| **Multi-space, multi-rate rate push** | required/recommended/optional | 5+ Spaces (3 Rooms, 1 Dorm and 1 Bed) and 2 Rates are mapped from the [Test Hotel](mews-api.md#test-environment) and pushed to [Update Availability](channel-manager-api.md#update-availability) endpoint. | CHM returns `{ "success": true}` and data are changed in extranet | blah blah blah |